server/lib/ssg.ts

```diff
@@ -0,0 +1,43 @@
+import { GetStaticPropsContext } from "next";
+import { i18n } from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import superjson from "superjson";
+
+import prisma from "@lib/prisma";
+
+import { appRouter } from "@server/routers/_app";
+import { createSSGHelpers } from "@trpc/react/ssg";
+
+/**
+ * Initialize static site rendering tRPC helpers.
+ * Provides a method to prefetch tRPC-queries in a `getStaticProps`-function.
+ * Automatically prefetches i18n based on the passed in `context`-object to prevent i18n-flickering.
+ * Make sure to `return { props: { trpcState: ssr.dehydrate() } }` at the end.
+ */
+export async function ssgInit<TParams extends { locale?: string }>(opts: GetStaticPropsContext<TParams>) {
```

Comment:
This function is taken from the pending #1053

Reply:
whats missing for this pending PR to go live?